src/main/java/org/worldcubeassociation/ui/ExportJSONAction.java

```diff
@@ -45,7 +46,7 @@ public void actionPerformed(ActionEvent aActionEvent) {
         }
 
         try {
-            String jsonFileName = "Results for " + getEnv().getCompetitionId() + ".json";
+            String jsonFileName = "Results (" + WorkbookAssistant.class.getPackage().getImplementationVersion() + ") for " + getEnv().getCompetitionId() + ".json";
```

Comment:
Great, thanks! I've updated https://github.com/cubing/wca-website/wiki/WCA-Competition-JSON-Format with a new `resultsProgram` field, and updated the `formatVersion` to `WCA Competition 0.3`.